config/ports.env

```diff
@@ -1,5 +1,5 @@
 METADATA_PORT=55500
-METADATA_URL=http://localhost:55000/dev.xml
+METADATA_URL=http://localhost:55500/dev.xml
```

Comment:
worth interpolating METADATA_PORT on this line?

Reply:
Can't interpolate without doing an eval somewhere :(